x-pack/solutions/security/plugins/security_solution/public/siem_migrations/rules/components/rule_details_flyout/tabs/translation/callout.tsx

```diff
@@ -9,14 +9,19 @@ import type { FC } from 'react';
 import React from 'react';
 import type { IconType } from '@elastic/eui';
 import { EuiCallOut } from '@elastic/eui';
+import type { RuleMigration } from '../../../../../../../common/siem_migrations/model/rule_migration.gen';
 import {
   RuleMigrationTranslationResultEnum,
   type RuleMigrationTranslationResult,
 } from '../../../../../../../common/siem_migrations/model/rule_migration.gen';
 import * as i18n from './translations';
 
+enum MappedTranslationResult {
+  MAPPED = 'mapped',
+}
+
 const getCallOutInfo = (
-  translationResult: RuleMigrationTranslationResult
+  translationResult: RuleMigrationTranslationResult | MappedTranslationResult
```

Comment:
"mapped" is not a translation result, this may confuse other devs. Consider calling this variable the `calloutMode` or just `mode` here.
And since the `RuleMigrationTranslationResult` used here is not a native enum, better use a union type:

`type RuleMigrationTranslationCallOutMode = RuleMigrationTranslationResult | 'mapped';`

and do the switch with strings:
```
 case 'full'
 case 'mapped'
 ...
```

Reply:
Done in https://github.com/elastic/kibana/pull/207789/commits/7818d19e9b72aac5d5f6649a6fc2e0eb92d1eff8